app/assets/scripts/components/connected/emergencies-table.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+import React from 'react';
+import { connect } from 'react-redux';
+import { PropTypes as T } from 'prop-types';
+import { Link } from 'react-router-dom';
+import ReactPaginate from 'react-paginate';
+import { DateTime } from 'luxon';
+
+import { environment } from '../../config';
+import { getEmergenciesList } from '../../actions';
+import { nope, commaSeparatedNumber as n } from '../../utils/format';
+import { get } from '../../utils/utils';
+
+import Fold from '../fold';
+import { showGlobalLoading, hideGlobalLoading } from '../global-loading';
+
+class EmergenciesTable extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      page: 1
+    };
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+
+  componentDidMount () {
+    this.requestResults();
+  }
+
+  componentWillReceiveProps (nextProps) {
+    if (this.props.list.fetching && !nextProps.list.fetching) {
+      hideGlobalLoading();
+    }
+  }
+
+  requestResults () {
+    showGlobalLoading();
+    this.props._getEmergenciesList(this.state.page);
+  }
+
+  handlePageChange (page) {
+    this.setState({ page: page.selected + 1 }, () => {
+      this.requestResults();
+    });
+  }
+
+  renderRow (rowData, idx, all) {
+    const disasterDate = rowData.disaster_start_date ||
+      rowData.start_date ||
+      rowData.created_at;
+
+    const date = disasterDate
+      ? DateTime.fromISO(disasterDate).toISODate() : nope;
+
+    const beneficiaries = get(rowData, 'appeals', []).reduce((acc, next) => {
+      return acc + next.num_beneficiaries;
+    }, 0);
+
+    const countries = get(rowData, 'countries', []).map(c => (
+      <Link className='link--primary' key={`c.iso`} to={`/country/${c.id}`}>{c.name}</Link>
+    ));
+
+    return (
+      <React.Fragment key={rowData.id}>
+        <tr>
+          <td data-heading='Date'>{date}</td>
+          <td data-heading='Name'>{get(rowData, 'name', nope)}</td>
+          <td data-heading='Disaster Type'>{get(rowData, 'dtype.name', nope)}</td>
+          <td data-heading='Total Affected'>{n(get(rowData, 'num_affected'))}</td>
+          <td data-heading='Beneficiaries'>{n(beneficiaries)}</td>
+          <td data-heading='Countries'>{countries.length ? countries : nope}</td>
+        </tr>
+      </React.Fragment>
+    );
+  }
+
+  render () {
+    const {
+      data,
+      fetched,
+      receivedAt,
+      error
+    } = this.props.list;
+
+    if (!receivedAt && !fetched) return null;
+
+    if (error) {
+      return (
+        <Fold title='Latest Alerts'>
+          <p>Oh no! An error ocurred getting the stats.</p>
+        </Fold>
+      );
+    }
+
+    if (!data.objects.length) {
+      return (
+        <Fold title='Latest Alerts'>
```

Comment:
Here and above you have "Latest Alerts" but below you have "Latest Emergencies". Is this correct?

Reply:
👍 they should all be "Latest Emergencies"